libraries/chain/wasm_eosio_validation.cpp

```diff
@@ -59,4 +59,19 @@ void globals_validation_visitor::validate( const Module& m ) {
             ("k", wasm_constraints::maximum_mutable_globals));
 }
 
+void maximum_function_stack_visitor::validate( const IR::Module& m ) {
```

Comment:
This looks like it will not calculate the runtime stack size, just the "static size of all functions".  Many of these functions may never be called.

Reply:
Yep, that's correct. I was trying to keep the PRs small and separate because big complex ones are harder to review and tend to languish.

This PR just makes sure that no single function consumes > 8192 bytes of stack size. A follow on PR will then add a runtime stack depth check. At that point maximum runtime usage will be bounded by maximum_func_local_bytes*maximum_stack_depth